src/components/DetailSidebar/DetailSidebar.tsx

```diff
@@ -14,14 +14,18 @@ import Repository from '../Repository/Repository';
 
 import { DetailContext } from '../../pages/Version';
 
-import { TitleListItem, TitleListItemText } from './styles';
+import { TitleListItem, TitleListItemText, PackageDescription, PackageVersion } from './styles';
 
 const renderLatestDescription = (description, version, isLatest: boolean = true): JSX.Element => {
   return (
-    <span>
-      <div>{description}</div>
-      {version ? <small>{`${isLatest ? 'Latest v' : 'v'}${version}`}</small> : null}
-    </span>
+    <>
+      <PackageDescription>{description}</PackageDescription>
```

Comment:
Coud you give me more context about this change? 🤔

Reply:
`div` is not a valid decendant of `p`, thus a warning in  the console. I refactored the DOM to be a `span` with a block display instead of a `div`

Reply:
Thanks for the clarification 🚀